tools/ssvm-aot/ssvmr.cpp

```diff
@@ -53,11 +57,119 @@ int main(int Argc, const char *Argv[]) {
   WasiMod->getEnv().init(Dir.value(), SoName.value(), Args.value(),
                          Env.value());
 
-  if (auto Result = VM.runWasmFile(InputPath, "_start")) {
-    return WasiMod->getEnv().getExitCode();
+  if (!Reactor.value()) {
+    // command mode
+    if (auto Result = VM.runWasmFile(InputPath, "_start")) {
+      return WasiMod->getEnv().getExitCode();
+    } else {
+      std::cout << "Failed. Error code : "
+                << static_cast<uint32_t>(Result.error()) << '\n';
+      return EXIT_FAILURE;
+    }
   } else {
-    std::cout << "Failed. Error code : "
-              << static_cast<uint32_t>(Result.error()) << '\n';
-    return EXIT_FAILURE;
+    // reactor mode
+    if (Args.value().empty()) {
+      std::cout << "reactor mode need function name.\n";
```

Comment:
`Reactor mode needs a given function name.`
Or
`A function name is required when reactor mode is enabled.`

Reply:
Done.